cluster/gce/configure-vm.sh

```diff
@@ -245,14 +245,34 @@ admission_control: '$(echo "$ADMISSION_CONTROL" | sed -e "s/'/''/g")'
 EOF
```

Comment:
BTW, I realized that `nounset` is, in fact, set throughout here. I think `create-salt-pillar` just doesn't trip it because each of those `$()` blocks is actually a subshell.